docs/quick_start/index.md

```diff
@@ -0,0 +1,118 @@
+
+!!! Prerequisites info
+    Prior to proceeding with this tutorial, ensure you have the following requirements installed: [kubectl](https://kubernetes.io/docs/tasks/tools/),
+    [K3D](https://k3d.io/#installation), [Helm](https://helm.sh/docs/intro/quickstart/), [Tilt](https://docs.tilt.dev/install.html), [Terraform](https://www.terraform.io/), [dotnet](https://dotnet.microsoft.com/en-us/download), [git](https://git-scm.com/).
+
+## Setting up the template
+
+To get started, begin by cloning the repository containing `lncdproj` template:
+
+```sh
+git clone git@github.com:leancodepl/exampleapp.git
+```
+
+Once the repository is cloned, navigate to the `exampleapp` directory. Here, install the `lncdproj` .NET template:
+
+```sh
+dotnet new install .
+```
+
+Then, you can run it in your preferred directory using the `dotnet new lncdproj` command. The `lncdproj` template requires two arguments: `--project-name`, specifying the name of the newly created project, and `--context`, indicating the name of the first bounded context used throughout the project (additional `--allow-scripts` flag is used to not display user prompts):
+
+```sh
+dotnet new lncdproj --allow-scripts Yes --project-name ProjectName --context ContextName
+```
+
+This template generates 3 directories:
+
+- `backend`: Contains the dotnet solution with template files.
+- `dev-cluster`: Holds the cluster configuration for local development.
+- `infrastructure`: Contains the configuration for infrastructure in production environments.
+
+## Setting up the local cluster
+
+Let's set up a local Kubernetes cluster, which is a vital part of our development environment. The `dev-cluster` directory contains configuration files for various services that will be deployed to our local Kubernetes cluster. These services include:
+
+- **PostgreSQL Database**: A relational database system for storing and managing application data.
+- **Blob Storage**: A service for storing large amounts of unstructured data, like images or videos.
+- **RabbitMQ**: A message broker that enables applications to communicate with each other asynchronously.
+- **Kratos**: An identity management system that handles user accounts, authentication, and authorization.
+- **Seq**: Log system designed for searching, analysis, and visualization of structured log data.
+- **Jaeger**: An open-source tool for tracing requests.
+- **Traefik**: A cloud-native application proxy that routes incoming requests to the appropriate microservices.
+- **OpenTelemetry**: A set of APIs, SDKs, and tools to generate, collect, and export telemetry data.
+- **Metabase**: Tool for visualizing and sharing data insights.
+
+To configure the local cluster, start by navigating to the `dev-cluster` directory. Once there, ensure that you've set the `sendgrid_api_key` in the `terraform.tfvars` file. Without this api key, the functionality for sending emails won't be available:
```

Comment:
I think it's worth at least to mention that it could be optionally set, as otherwise user will get few annoying prompts for   `sendgrid_api_key` during `terraform apply`.

Reply:
I wonder if we should set defaults there in the first place.